doc/operation_guides/manual/upgrade-guide.md

```diff
@@ -0,0 +1,150 @@
+# OpenProject 4.0 to OpenProject 4.1 Debian/Ubuntu Upgrade Guide
+
+This guide describes the upgrade process from OpenProject 4.0 to 4.1 on Debian 7.7 and
+Ubuntu 14.04 LTS step by step.
+
+Note: We strongly recommend to update your OpenProject installation to the latest
+available 4.0 version (currently 4.0.9), before attempting an update to 4.1.
+
+
+## Preparation
+
+* Backup your current Openproject installation. Typically you should backup the attachment
+  folder of your installation, the subversion repositories (if applicable) and your database.
+  For more information please have a look at our [backup guide](backup-guide.md)
+
+* Before Upgrading, check that all the installed OpenProject plugins support the new
+  OpenProject version. Remove incompatible plugins before attempting an upgrade. Stop
+  the OpenProject Server. You may even add a maintenance page, if you feel comfortable
+  with that.
+
+* If you run the worker process with a cronjob, disable the cronjob temporarily.
+* Stop the (delayed\_job) worker process. In case you run the woker process through
+  `RAILS_ENV=production bundle exec script/delayed_job start`, execute the following:
+  `RAILS_ENV=production bundle exec script/delayed_job stop`.
+
+## Update your system
+
+[root@debian]# apt-get update
+[root@debian]# apt-get upgrade
```

Comment:
I think you forgot to mark that section as code. Is is displayed oddly as rendered markdown.